operations/configuration_management/Target_Ansible_Tasks_for_Image_Customization.md

```diff
@@ -18,5 +20,9 @@ when: "{{ cray_cfs_image | default(false) }}"
 
 If a default is not provided, any playbooks or roles will not be runnable outside of the CFS Ansible Execution Environment \(AEE\) without the user specifying `cray_cfs_image` in the vars files or with the Ansible extra-vars options.
 
+CFS automatically sets this variable in the hosts/01-cfs-generated.yaml file for all sessions. When the session target is image customization, it sets `cray_cfs_image` to `true`; otherwise, it is `false`.
```

Comment:
```suggestion
CFS automatically sets this variable in the `hosts/01-cfs-generated.yaml` file for all sessions. When the session target is image customization, it sets `cray_cfs_image` to `true`; otherwise, it is `false`.
```